lib/vdsm/storage/managedvolume.py

```diff
@@ -297,3 +303,25 @@ def _remove_udev_rule(vol_id):
     except Exception:
         log.exception(
             "Failed to remove udev rule for volume %s", vol_id)
+
+
+def _add_run_link(vol_id, path):
+    _create_run_dir()
+    run_path = os.path.join(VOLUME_LINK_DIR, vol_id)
+    os.symlink(path, run_path)
+
+    return run_path
+
+
+def _remove_run_link(vol_id):
```

Comment:
Looks like _silent_remove_run_link(), since it consume the error.

Same for _remove_udev_rule().

Using bare "except:" is good when you want to do something in error
flow and raise the original error, as we do in _add_udev_rule().